keras_nlp/models/task_test.py

```diff
@@ -64,11 +64,11 @@ def test_summary_with_preprocessor(self):
         preprocessor = SimplePreprocessor()
         model = SimpleTask(preprocessor)
         summary = []
-        model.summary(print_fn=lambda x, line_break: summary.append(x))
+        model.summary(print_fn=lambda x, line_break=False: summary.append(x))
```

Comment:
You should be able to skip `line_break` entirely, no?

`model.summary(print_fn=lambda x: summary.append(x))`

Reply:
I also think we should remove `line_break` but when I do that, it goes to [this branch](https://github.com/keras-team/keras/blob/d7f4605935614150863e1aa8508702599b28601a/keras/utils/summary_utils.py#L363-L364) and gives the following error:
```
TypeError: TestTask.test_summary_with_preprocessor.<locals>.<lambda>() got an unexpected keyword argument 'line_break'
```
I should figure out why  [this branch](https://github.com/keras-team/keras/blob/d7f4605935614150863e1aa8508702599b28601a/keras/utils/summary_utils.py#L363-L364) runs when I remove `line_break`.